recipes/src/main/resources/quarkus-updates/core/3.2.yaml

```diff
@@ -23,6 +23,33 @@ recipeList:
   - org.openrewrite.properties.ChangePropertyKey:
       oldPropertyKey: quarkus.transaction-manager.object-store-directory
       newPropertyKey: quarkus.transaction-manager.object-store.directory
+- org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.opentelemetry.enabled
+      newPropertyKey: quarkus.otel.enabled
+- org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.opentelemetry.tracer.enabled
+      newPropertyKey: quarkus.otel.traces.enabled
+- org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.opentelemetry.tracer.suppress-non-application-uris
+      newPropertyKey: quarkus.otel.traces.suppress-non-application-uris
+- org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.opentelemetry.tracer.include-static-resources
+      newPropertyKey: quarkus.otel.traces.include-static-resources
+- org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.opentelemetry.tracer.sampler
+      newPropertyKey: quarkus.otel.traces.sampler
+- org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.opentelemetry.tracer.sampler.ratio
+      newPropertyKey: quarkus.otel.traces.sampler.arg
```

Comment:
@brunobat this one looks odd to me. Is it a 1-1 mapping?

I based my work on https://github.com/quarkusio/quarkus/pull/32050/files#diff-ce0b5beb6a1b039064cc19f31a98739c58721ab1c51f90f4fbfe3568c7eb7f5cR17-R28 .

Reply:
Those properties showed up previously. The full explanation is here: https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0#extension-re-write 
The sampler properties were re-writen and they don't map 1-1 with the older ones.

Reply:
If they don't map 1-1 and we can't rewrite them, we shouldn't include them. It's only this one that I should remove or there are others that won't work properly if they are rewritten?

Reply:
Only the sample related ones... The others should be fine.